code/helpers/overlays.dm

```diff
@@ -0,0 +1,13 @@
+/atom/proc/create_overlay(obj/item/equippable/from)
+	var/image/overlay = new
+	overlay.icon = from.worn_icon
+	overlay.icon_state = from.worn_icon_state
```

Comment:
it would probably be better to just have a proc for getting an image from icon + icon state